blocks/image_and_text/_imageandtext.pug

```diff
@@ -0,0 +1,11 @@
+- $image_url = get_field('imgtxtimage')['url']
+- $element_class = get_field('imgtxtorder') == 'imgtext' ? '' : 'imageandtext--reverse'
+
+.wrapper
```

Comment:
From the component isolation POW I think it's not correct to include its BEM root inside a generic class like `.wrapper` and generally speaking inside anything.

In order to have a real separations of concern and responsibility this should be

```pug
.imageandtext
  .imageandtext_wrapper
```

with realted SASS wrote in the right place. Or, as a better solution, let the developer manage the outer context inside the hosting template.